radis/lbl/broadening.py

```diff
@@ -2251,9 +2251,7 @@ def _broaden_lines(self, df):
 
                 # Raise performance warning if the chunks are bigger
                 # than the number of lines.
-                try:
-                    assert N < len(df)
-                except AssertionError:
+                if N >= len(df):
                     self.warn(
                         "There are currently more chunks ({0:.3e}) than lines ({1:.3e}),".format(
```

Comment:
Here the message is not very helpful to the user. They'll say "Sure, so what?" 
Suggest to increase chunksize to a reasonable value (similar to the other comment)

Reply:
Hello, the following lines have the same message as the previous commit, should I add something else too?

Reply:
yes